docs/src/examples/FirstAnimation.tsx

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import { Button, StyleSheet, View } from 'react-native';
-import Animated, { useSharedValue, withSpring } from 'react-native-reanimated';
+import Animated, { SharedValue, useSharedValue, withSpring } from 'react-native-reanimated';
```

Comment:
I see that you import type `SharedValue` many times but you don't use it. Is it intentionally?